cli/types/cypress-npm-api.d.ts

```diff
@@ -184,13 +184,24 @@ declare namespace CypressCommandLine {
     attempts: AttemptResult[]
   }
 
+  interface Timings {
+    lifecycle: number
+    test: {
+      hookId?: any
```

Comment:
This should be `string` - that said, these are internal properties that is not part of our public API, I am not sure we should be defining types for this? It could change at any moment.

Alternatively, they could be marked as `@private`, but rather than having an internal property `hookId` that is nullable and `any`, maybe we should remove these. Considering this is from the Mocha test runner, and they can change this at any time - we do not control it.

Reply:
WDYT @mike-plummer ?

Reply:
Good callout @lmiller1990 , from the writeup it seems like `timings` and `failedFromHookId` aren't actually being relied on but were added here because the fields were seen as "present" when inspecting the result. Probably best to remove them both unless a specific external need for them is identified, especially since the internal mocha-derived types aren't typed very well as-is.

Reply:
Done.